.github/workflows/test-e2e-fork.yml

```diff
@@ -44,7 +44,8 @@ jobs:
               ref: process.env.ref
             });
 
-            const check = checks.check_runs.filter(c => c.name === 'e2e-test');
+            // update the 'check-external-pr' check run
```

Comment:
I see that we set it based on the result of the triggered workflow. While this is fine, I think it's a bit inaccurate. If we managed to trigger the workflow and it completed, then we've checked the external PR. So we can always set it to `success` as the conclusion.

Reply:
Makes sense. Update it to always set to to `'success'`